### ISSUE TITLE
ETHDenver schedule

### ISSUE THREAD

**Issue by ryscheng** [opensource-observer/oso#733]
As per our conversation today,

we probably want to balance:
- Coworking together as a team, hacking on stuff together
- Team planning
- Talks / panels that we're apart of
- Our community event https://github.com/opensource-observer/oso/issues/720
- Talking to specific partners

Let's put together a rough schedule of events and things we want to do.

**Comment by ryscheng** [comment]
@ccerv1 do you have lists of events that you plan on going to?
Let's collaborate so we make sure we make time for team time too.

**Comment by ccerv1** [comment]
@ryscheng FYI:

I've updated the team calendar with relevant events. There are only two that are "musts":

- The Gitcoin x Electric Capital ecosystem growth summit (Friday afternoon)
- Our event (Saturday afternoon)

I've added a few others marked as optional. I've scanned a few side events lists and will continue to look out for others. But what I might suggest is we spend Thur - Fri - Sat morning as team time and do events / meetings after lunch.
